charge.go

```diff
@@ -551,6 +551,10 @@ type ChargePaymentMethodDetailsStripeAccount struct {
 type ChargePaymentMethodDetailsWechat struct {
 }
 
+// ChargePaymentMethodDetailsWechatPay represents details about the WechatPay PaymentMethod.
+type ChargePaymentMethodDetailsWechatPay struct {
+}
```

Comment:
missing `Fingerprint` and `TransactionID`

Reply:
Fixed